### ISSUE TITLE
Adjust schedules for view and data refresh DAGs

### ISSUE THREAD

**Issue by stacimc** [WordPress/openverse-catalog#484]
## Current Situation

Currently the `*_data_refresh` run `@weekly`, so UTC 0 Sunday morning, and the associated `refresh_*_view_data` run at UTC 00:00 every day except the first of the month.

## Suggested Improvement

Since we want the view refresh to run before the data refresh, it might be better to run both sets of DAGs weekly with an offset to allow the view refresh to finish before the data refresh begins. For example, we could switch the view refreshes to `@weekly` so they run at default Sunday UTC 00:00, and schedule the data refreshes to run weekly on _Mondays_, so they begin during working hours and having given the view refresh time to complete.

## Additional context

In future iterations, we plan on consolidating dependencies into a single data_refresh DAG.

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.

**Comment by stacimc** [comment]
TL;DR: I don't think we should act on this until #453, which will resolve the scheduling much more effectively by coupling the popularity calculation/matview refresh/data refresh tasks in Airflow.

Currently, we run `refresh_all_*_popularity_data` (which calculates popularity data _and_ refreshes the matview) on the first day of each month, and `refresh_*_view_data` (which _only_ refreshes the matview) on all other days. The data refresh itself is run weekly, each Sunday at UTC 0:00.

The matview appears to only be used by the data refresh, so it's not really necessary for it to run every day -- we _could_ just run it weekly, before the data refresh. To my knowledge though, we can't easily express this in a cron expression while also avoiding runs on the first of the month. Of course there are a number of ways to get around this (checking the date within the task, setting up dependencies between the DAGs) but anything more complicated than updating the schedule interval doesn't seem like a worthwhile investment when #453 is already planned for the next milestone.

**I think the simplest way forward is to turn off the data refresh DAGs temporarily.** This is only because this coming Sunday is May 1st, which happens to be both the first of the month and the first of the week -- so the popularity data calculations and data refreshes will be triggered simultaneously. **We should wait for the popularity calculations to complete early next week before turning the data refreshes back on.** We then have until January 2023 before the first of the month falls on a Sunday again, in order to complete #453 😄

**Comment by AetherUnbound** [comment]
These are great points Staci! And what an unfortunate calendar collision 😅 Per a [discussion in the Make WordPress slack](https://wordpress.slack.com/archives/C02012JB00N/p1650986008294839), we're going to close this in favor of #453. We believe that work will cover the problems expressed in this issue.
